index.js

```diff
@@ -39,6 +39,7 @@ function RegClient (config) {
   if (typeof this.config.retry.factor !== 'number') this.config.retry.factor = 10
   if (typeof this.config.retry.minTimeout !== 'number') this.config.retry.minTimeout = 10000
   if (typeof this.config.retry.maxTimeout !== 'number') this.config.retry.maxTimeout = 60000
+  if (typeof this.config.maxSockets !== 'number') this.config.maxSockets = 50
```

Comment:
Yeah, I expect we'll want the npm side of this patch to pass through a value and to have a default there too, since this is the sort of thing where the best value is likely heavily impacted by network conditions.